css/index.css

```diff
@@ -0,0 +1,152 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.main-page {
+  display: flex;
+  flex-flow: column;
+  max-width: 100%;
+}
+
+.interface {
+  background-image: url("../assets/background.jpg");
+  width: 100%;
+  height: 550px;
+  background-position: bottom;
+  background-repeat: no-repeat;
+}
+
+.nav-bar {
+  display: flex;
+  height: 35  px;
+  max-width: 900px;
+  border-radius: 6px;
+  background-color: #707070;
+  margin: 80px auto;
+  justify-content: space-around;
+  flex-flow: row;
+  text-align: center;
+}
+
+.nav-list {
+  display: inline-flex;
+  width: 105px;
+  align-items: center;
+  border-right: 1px solid #606060;
+  height: 35px;
+  justify-content: center;
+  padding-top: 12 px;
+}
+
+a {
+  text-decoration: none;
+}
+
+footer a {
+  color: #0ebcf1;
+}
+
+.nav-list a {
+  color: seashell;
+  font-size: 15px;
+  width: 15px;
```

Comment:
increase the `width` to `45px` to align all elements on center, the 'Suport' element is not centered

Reply:
thank you. i changed your suggestions